### PR TITLE
Embed Applier interface in kubectlClient interface 

### PR DESCRIPTION
**Pull request by darkowlzz**
Rename Applier interface method from `Applyx` to `Apply` and embed
the Applier interface in the `kubectlClient` interface.

### CODE REVIEW

**Comment by atoato88** [comment]
/lgtm
Nice catch!! and because of this, we can use this interface efficient.

**Comment by atoato88** [comment]
@darkowlzz 
It's good to assign the approver to this PR for smooth merging process.
Please refer comments above from k8s-ci-bot in this PR.

**Comment by darkowlzz** [comment]
Thanks @atoato88 .

/assign @mengqiy

File: pkg/patterns/declarative/reconciler.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ type Reconciler struct {
 }
 
 type kubectlClient interface {
-	Apply(ctx context.Context, namespace string, manifest string, validate bool, args ...string) error
+	applier.Applier
```
Comment (justinsb) [inline pkg/patterns/declarative/reconciler.go:68]: This seems reasonable.  I do wonder if there's any reason to keep the `kubectlClient` interface, if we have the exported applier.Applier interface...

**Comment by justinsb** [comment]
Thanks @darkowlzz 

/approve
/lgtm

File: pkg/patterns/declarative/reconciler.go (hunk 1 above)
Comment (atoato88) [reply pkg/patterns/declarative/reconciler.go:68]: Agree with your comment, we can use `Applier` interface instead of `kubectlClient` interface.

**Pull request merged**